server/core/src/main/scala/org.apache.daffodil.debugger.dap/DAPodil.scala

```diff
@@ -194,7 +194,7 @@ class DAPodil(
           case Left(errors) =>
             state.set(DAPodil.State.FailedToLaunch(request, errors, None)) *>
               Logger[IO].warn(show"error parsing launch args: ${errors.mkString_(", ")}") *> session
-              .sendResponse(request.respondFailure(Some(show"error parsing launch args: ${errors.mkString_(", ")}")))
+                .sendResponse(request.respondFailure(Some(show"error parsing launch args: ${errors.mkString_(", ")}")))
```

Comment:
ditto. Revert just whitespace changes because they cause trouble trying to understand the version changes which is what this PR is about.

Reply:
all the changes in this file appear to be just whitespace

Reply:
Same issue https://github.com/apache/daffodil-vscode/pull/219#discussion_r918337631 will look into ignoring spacing items

Reply:
Yeah, if you can just pull these from this commit that would be good. A subsequent commit of just code-formatting fixups is fine of course, particularly if it brings the code into coherence with a standard formatting.

Reply:
@mbeckerle It seems scalafmt doesn't allow us just to ignore certain lines. So I think either the PR has to have to formatting fixes in it or I revert the formatting stuff and make separate PR for it.

Reply:
See https://scalameta.org/scalafmt/docs/configuration.html#-format-off for a solution